parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -529,6 +529,17 @@ try
             Remove-Item $kubeConfigFile
         }
 
+        # Enable FIPS-mode
+        $fipsEnabled = [System.Convert]::ToBoolean("{{ FIPSEnabled }}")
+        if ( $fipsEnabled ) {

```

Comment:
Could we add a function in `windowsconfigfunc.ps1` and call the function here? We will introduce new Windows version soon and it may need to check os version and then set different configurations.
1. Put `$fipsEnabled = [System.Convert]::ToBoolean("{{ FIPSEnabled }}")` before `try`
2. Call `Set-FIPS -Enable $fipsEnabled` in line 532

Reply:
Move code to set the registry to a separate function, and move '$fipsEnabled = [System.Convert]::ToBoolean("{{ FIPSEnabled }}") ' before try according to this comment.